src/java.base/linux/classes/sun/nio/ch/EPollSelectorImpl.java

```diff
@@ -86,7 +86,7 @@ private static class MoveToCheckpointThread extends Thread {
         public void run() {
             try {
                 selector.select(1);
-            } catch (IOException e) {
+            } catch (IOException | RuntimeException e) {
```

Comment:
I am worried that suppressing exceptions beyond `ClosedSelectorException` could hide a future underlying problem. Do you have a strong reason to suppress also NPEs, for instance?